src/bin/common.rs

```diff
@@ -175,6 +175,14 @@ pub fn parse_cli() -> Result<CliOptions, CliError> {
         .takes_value(true)
         .default_value("6")
     )
+    .arg(
+      Arg::with_name("SCENE_CHANGE_DETECTION_SPEED")
+        .help("Speed level for scene-change detection, 0: best quality, 1: speed-to-quality trade-off, 2: fastest mode\n\
+          [default:  0 for s0-s6, 1 for s7-s9, 2 for s10]")
+        .long("scd_speed")
+        .takes_value(true)
+        .default_value("2")
```

Comment:
I think we want to remove the default value here, because by default we want it set according to the speed level.

Reply:
It is set according to the speed level by default on line 607.
`cfg.speed_settings.fast_scene_detection` changes on lines 609-617 only when the user specifies a value of scene detection speed.

Reply:
Have we tested that? My concern was that `matches.occurrences_of("SCENE_CHANGE_DETECTION_SPEED")` would pick up the default value of "2" specified here. I'd like to verify that's not the case.

Reply:
matches.occurrences_of("SCENE_CHANGE_DETECTION_SPEED") is 0 if no flags are passed. If that's your concern.